apps/examples/board_specific_demo/ameba_mipi_demo/Kconfig_ENTRY

```diff
@@ -0,0 +1,3 @@
+config ENTRY_MIPI
```

Comment:
Let's add `AMEBA` at the name

Reply:
Dear @sunghan-chang 
Added as suggested.